packages/react/src/components/Footer/Footer.js

```diff
@@ -4,7 +4,7 @@ import classnames from 'classnames';
 import Link from '../Link';
 import Wrapper from '../Wrapper';
 // import Icon from '../Icon';
-import { FooterExternal } from './FooterExternal';
+import {FooterExternal} from './FooterExternal';
```

Comment:
Would it be better to provide `<Footer />` and `<FooterExternal />` as two components?

Reply:
Sure, I can work on that